config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "SERVER_PORT": 9001,
+    "CLIENT_WEBSERVER_PORT": 9002,
+    "SEARCH_ID_TIMEOUT": 120000,
+    "SEARCH_ID_AND_FEATURES_TIMEOUT": 300000,
+    "DAYS_OF_SEARCH": 259200000,
+    "INTERVAL_SEARCH_DAYS": 600000,
```

Comment:
What do these values mean? For example, one of them can be interpreted as "Search for 259200000 days" (?)

Reply:
I can't add comments in a json file, should i change it to a .js file?

Reply:
I will add `_IN_MILLISECONDS` and should be fine

Reply:
Ideally the field names should be descriptive enough not to require comments.